notebooks/test_distributions.ipynb

```diff
@@ -0,0 +1,975 @@
+{
```

Comment:
<p>Why are we limiting to only common distributions? Can we try it over all the distributions to see which offers the the lowest sum square error?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/optimal-stopping-point-tool/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yes, can add that here.